### DISCUSSION TITLE
is the cropperComponent limited by image source size?

### DISCUSSION THREAD

**Discussion by AlbertLiu1974** [CropperBlazor/Cropper.Blazor#324] [General]
Hello, 

  I got problem when give a more larger size image as attached image about 3.3MB 14000x9940. It seems not showing. Kindly help. 

![image](https://github.com/CropperBlazor/Cropper.Blazor/assets/152601781/a483c48a-804a-43e2-b999-2c434bd5e3ea)

Best regards,
Albert

**Comment by AlbertLiu1974** [comment]
I felt the chrome memory may have limited on 2G.  when I change the pixel size to smaller.. it works.  is it a bug or can we config some setup to overcome this issue?

**Reply by MaxymGorn** [reply]
Hi @AlbertLiu1974. I have no clue in this moment. Could you provide more details?

**Reply by AlbertLiu1974** [reply]
I guess it's related with the uploading image numbers. when I upload more than 100 pages of PDF... Chrome said out of memory.  I am thinking to chunk size of upload method. 2G seems the chrome default max memory size?

Br, Albert

**Reply by MaxymGorn** [reply]
> I guess it's related with the uploading image numbers. when I upload more than 100 pages of PDF... Chrome said out of memory. I am thinking to chunk size of upload method. 2G seems the chrome default max memory size?
> 
> Br, Albert

Are you cleaning up extra blobs? Something like this: `CropperComponent?.RevokeObjectUrlAsync(Src);`. Note: in the future, this will be moved to a separate service.
